docs/application/dotnet/guides/nui/ui-components.md

```diff
@@ -29,6 +29,8 @@ The following table lists the available UI components:
 | [TextLabel](./textlabel.md)          | A text label that renders a short text string. | `TextLabel`                |
 | [Toast](./nui-components/Toast.md) | A toast is used to provide simple messages. | `Toast`                |
 | [VideoView](./videoview.md)          | A video view that controls and displays video playback. | `VideoView`                 |
+| [CheckBox](./nui-components/CheckBox.md) | A CheckBox that can set selected/unselected status when user select it. | `CheckBox`                |
+| [RadioButton](./nui-components/RadioButton.md) | A RadioButton that can set selected/unselected status when user select it. | `RadioButton`                |
```

Comment:
Request you to please follow alphabetical order for the content in the table. Please add "Checkbox" after "Button" and "RadioButton" after "Progress."

Also, suggest rephrasing to:

A CheckBox that can set selected or unselected status when user selects it.


A RadioButton that can set selected or unselected status when user selects it.

Please verify

Reply:
fixed